packages/instantsearch-codemods/package.json

```diff
@@ -1,9 +1,24 @@
 {
   "name": "instantsearch-codemods",
+  "description": "Codemods for Algolia InstantSearch libraries",
   "version": "0.0.0",
```

Comment:
this version is fine, we'll see what the release process bumps it to and then update if needed, 1.0.0 sounds good, but it's a codemod so the version doesn't matter much